src/page/order-summary.page.ts

```diff
@@ -4,7 +4,7 @@ export class OrderSummaryPage {
   private orderStatus: ElementFinder;
 
   constructor () {
-    this.orderStatus = $('#center_column > div > p > strong');
+    this.orderStatus = $('.cheque-indent');
```

Comment:
Did you validate this selector was unique?

Reply:
Yes sir, both bankwire and cheque-indent when i was inspecting the page i validate if the value was unique.